### PR TITLE
(Feature) Add dropdown for selecting mining key, add limit notification

### LINKED ISSUE #51: Get rid of sokol branch. Provide Dynamic Loading of addresses based on Metamask netId

### PR DESCRIPTION
**Pull request by rstormsf**
(Feature#1) Dropdown of mining keys
As a validator, I'd like to select existing mining key from PoA contract. 
If we are going to add new mining key, I should be able to provide new value.

(Feature#2) Limits:
As a validator, I'd like to know how many ballots I can create having already active ballots
  
(Fix#3) Load addresses based on ChainId
Fixes #51

### CODE REVIEW

File: src/stores/ContractsStore.js [hunk 1]
```diff
@@ -33,15 +33,18 @@ class ContractsStore {
 	@observable keysBallotThreshold;
 	@observable minThresholdBallotThreshold;
 	@observable proxyBallotThreshold;
+	@observable validatorLimits;
```
Comment (vbaranov) [inline src/stores/ContractsStore.js:35]: Duplicate of https://github.com/poanetwork/poa-dapps-voting/pull/50/files#diff-795e7edef556435c55ff76927dc5369eR22

File: src/contracts/addresses.js [hunk 2]
```diff
@@ -1,8 +1,38 @@
...
+    METADATA_ADDRESS: '0xce9ff1123223d13672cce06dd073d3749764daa6',
+    POA_ADDRESS: '0x8bf38d4764929064f2d4d3a56520a76ab3df415b',
+}
+
+module.exports = (netId) => {
+    switch (netId) {
+        case '77':
+            return SOKOL_ADDRESSES
+        case '99':
+            return local
```
Comment (vbaranov) [inline src/contracts/addresses.js]: should CORE_ADDRESSES be here?

File: src/getWeb3.js [hunk 3]
```diff
@@ -29,6 +29,7 @@ let getWeb3 = () => {
               errorMsg = constants.WRONG_NETWORK_MSG
               console.log('This is an unknown network.', netId)
           }
+          document.title = `${netIdName} - POA governance dApp`
```
Comment (vbaranov) [inline src/getWeb3.js]: Let's rename to 
`${netIdName} - POA Network Governance DApp`
as suggested here https://github.com/poanetwork/poa-dapps-voting/pull/31/files/ab1761adf93cfd81c999fb66f1ec9a9ef8567e83#r158846016

File: src/stores/ContractsStore.js (hunk 1 above)
Comment (rstormsf) [reply src/stores/ContractsStore.js:35]: 👍

File: src/contracts/addresses.js (hunk 2 above)
Comment (rstormsf) [reply src/contracts/addresses.js]: you are right!

File: src/getWeb3.js (hunk 3 above)
Comment (rstormsf) [reply src/getWeb3.js]: ok

**Review by vbaranov — APPROVED** [review]

**Pull request merged**